deploy/lib/buildAndPushContainers.js

```diff
@@ -68,6 +68,9 @@ module.exports = {
     const containerNames = Object.keys(this.containers);
     const promises = containerNames.map((containerName) => {
       const container = this.containers[containerName];
+      if (container["directory"] == undefined) {
```

Comment:
```suggestion
      if (container["directory"] === undefined) {
```
The double equals == operator performs type coercion, meaning it converts the operands to the same type before making the comparison. This can sometimes lead to unexpected results, particularly when comparing values that may be falsy.